tests/libnet/ipv6.go

```diff
@@ -53,3 +53,13 @@ func ConfigureIPv6OnVMI(vmi *v1.VirtualMachineInstance) error {
 
 	return nil
 }
+
+func WithIPv6(loginToFactory console.LoginToFactory) console.LoginToFactory {
```

Comment:
I find it counter intuitive and a bit ugly to do this here. The two actions are not related at all, one they merit two actions at the caller. e.g:
```
tests.WaitUntilVMIReady(vmi, console.LoginToCirros)
configureIPv6OnVMI(vmi)
```

But, if it will bite me in the near future we can just change it. And as mentioned already, this is an intermediate solution until dynamic IPv6 is supported and the whole IPv6 configuration through the console will be dropped.

Reply:
As stated offline, we are going to continue with this solutions since is temporal and we need all this to fix SRIOV while dynamic ipv6 lands.